src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -2062,6 +2062,11 @@ private Field DetermineMinimalOptionalField(Node nd)
             }
         }
 
+        private static bool IsAttributeOrModifiersList(Field f)
```

Comment:
>IsAttributeOrModifiersList [](start = 28, length = 26)

could be local function?

Reply:
It will prevent conflicts with features/local-function-attributes to leave it as is. Maybe we can take care of it once one or the other feature branch is merged to master?